app/assets/javascripts/components/bulk-edit-measures.js

```diff
@@ -308,6 +315,20 @@ $(document).ready(function() {
       },
       measuresUpdated: function() {
         DB.insertOrReplaceBulk(this.search_code, this.measures);
+      },
+      selectAllHasChanged: function(value) {
+        this.selectedAllMeasures = value;
+      },
+      measuresRemoved: function(removedMeasures) {
+        var self = this;
+        removedMeasures.forEach(function(removedMeasure) {
+          var index = self.measures.indexOf(removedMeasure);
+          self.measures.splice(index, 1);
```

Comment:
@rusllonrails ATM we're removing the measures from the collection (so that they're removed from webDB and they're not rendered in the table).

however maybe we want to add a flag instead to these measures so in the backend we know when these have been removed